src/app/overview/overview.component.html

```diff
@@ -1,27 +1,55 @@
-<div class="col-md-12" style="overflow-y: scroll;height: 900px;">
+<div class="col-md-12 " style="overflow-y: scroll;height: 88vh;padding-bottom: 10px;font-family: 'Lato', sans-serif;">
```

Comment:
don't put the font family it will auto get the default one

Reply:
removed :+1: